### PR TITLE
For #23829 - Update private theme design token based on the latest UX changes

### LINKED ISSUE #23829: Update private theme design token based on the latest UX changes

### PR DESCRIPTION
**Pull request by gabrielluong**
Fixes #23829

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -131,37 +131,119 @@
...
+    <color name="fx_mobile_private_layer_color_4_start" tools:ignore="UnusedResources">@color/photonPurple70</color>
+    <!-- Homescreen background, Toolbar -->
+    <color name="fx_mobile_private_layer_color_4_center" tools:ignore="UnusedResources">@color/photonViolet80</color>
+    <!-- Homescreen background, Toolbar -->
+    <color name="fx_mobile_private_layer_color_4_end" tools:ignore="UnusedResources">@color/photonInk05</color>
+    <!-- App Bar Top (edit), Text Cursor, Selected Tab Check -->
+    <color name="fx_mobile_private_layer_color_accent" tools:ignore="UnusedResources">@color/photonViolet40</color>
+    <!-- Selected tab -->
+    <color name="fx_mobile_private_layer_color_accent_nonopaque" tools:ignore="UnusedResources">@color/photonViolet50A32</color>
+    <color name="fx_mobile_private_layer_color_scrim" tools:ignore="UnusedResources">@color/photonDarkGrey05A45</color>
```
Comment (MozillaNoah) [inline app/src/main/res/values/colors.xml:144]: is this missing a comment?

File: app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -131,37 +131,119 @@
     <color name="fx_mobile_private_layer_color_2" tools:ignore="UnusedResources">@color/photonInk50</color>
     <!-- Search -->
     <color name="fx_mobile_private_layer_color_3">@color/photonInk50</color>
+    <!-- Homescreen background, Toolbar -->
+    <color name="fx_mobile_private_layer_color_4_start" tools:ignore="UnusedResources">@color/photonPurple70</color>
```
Comment (MozillaNoah) [inline app/src/main/res/values/colors.xml:135]: are the start/center/end here used for a gradient of some sort?

Comment (gabrielluong) [reply app/src/main/res/values/colors.xml:135]: Ya, it's a gradient

File: app/src/main/res/values/colors.xml (hunk 1 above)
Comment (gabrielluong) [reply app/src/main/res/values/colors.xml:144]: Screen Shot 2022-02-22 at 4 27 43 PM

Nope

File: app/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -131,37 +131,119 @@
...
+    <!-- Icon inverted (on color) -->
+    <color name="fx_mobile_private_icon_color_oncolor" tools:ignore="UnusedResources">@color/photonLightGrey05</color>
+    <!-- Information -->
+    <color name="fx_mobile_private_icon_color_information" tools:ignore="UnusedResources">@color/photonBlue30</color>
+    <!-- Icon button -->
+    <color name="fx_mobile_private_icon_color_button" tools:ignore="UnusedResources">@color/photonLightGrey05</color>
+    <color name="fx_mobile_private_icon_color_warning" tools:ignore="UnusedResources">@color/photonRed20</color>
+    <!-- Warning icon on Secondary button -->
+    <color name="fx_mobile_private_icon_color_warning_button" tools:ignore="UnusedResources">@color/photonRed70</color>
+    <color name="fx_mobile_private_icon_color_accent_violet" tools:ignore="UnusedResources">@color/photonViolet20</color>
```
Comment (MozillaNoah) [inline app/src/main/res/values/colors.xml:224]: do the accent colors here need an info comment?

**Review by MozillaNoah — APPROVED** [review]

**Pull request merged**